Loop/Managers/LoopDataManager.swift

```diff
@@ -992,7 +992,7 @@ extension LoopDataManager {
 
         let retrospectiveStart = lastGlucoseDate.addingTimeInterval(-type(of: retrospectiveCorrection).retrospectionInterval)
 
-        let earliestEffectDate = Date(timeInterval: .hours(-24), since: now())
+        let earliestEffectDate = Date(timeInterval: .hours(-48), since: now())
```

Comment:
Proposed change to see a week of data in the table (need this and the next ones:
```
        let earliestEffectDate = Date(timeInterval: .hours(-7*24), since: now())
```

The data store contains 7 days of data. Perhaps this should be a variable tied to the days of storage parameter. (not sure where that is - just remember that it exists)

Reply:
Making this 7 days, makes the app very slowly on iPhone 11. It took a good 30-60sec before the UI got updated (aka the charts got filled with data). Maybe we should find a middle ground?